### ISSUE TITLE
Review Dropdown Accessibility

### ISSUE THREAD

**Issue by StephenRiosDev** [AlaskaAirlines/auro-formkit#453]
### Description

Assess accessibility of the Dropdown component. Ensure it meets all W3C standards and that the screen reader functions as expected.

W3C example: https://www.w3.org/WAI/ARIA/apg/patterns/combobox/examples/combobox-select-only/

### Exit criteria (To be determined by the Auro team)

Dropdown should meet or exceed all W3C 2.0 AA criteria.

**Comment by blackfalcon** [comment]
The floor-wide standard is to meet or exceed WCAG 2.2 Level AA. Please see [Alaska Airlines Digital Accessibility and Inclusion Policy](https://wiki.devtools.teamaag.com/standards/accessibility/digital-accessibility-policy) for more information.

**Comment by sun-mota** [comment]
No a11y violation found in layoutSupport
